app/views/admin/whats_new/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :page_title, t('whats_new.title') %>
+<% content_for :title, t('whats_new.title') %>
+<% content_for :title_margin_bottom, 0 %>
+
+<%
+  sections = [
+    t('whats_new.introduction'),
+    t('whats_new.recent_changes'),
+    t('whats_new.guidance'),
+  ]
+%>
```

Comment:
It looks like a premature optimisation making this into an array. I'd remove it.

Reply:
I don't think it's premature as we already know the `upcoming changes` section is coming in.

If we dismantle the array and have separate sections for each thing, it'll just mean each section will just duplicate code.

We also use this loop to map the content list (https://github.com/alphagov/whitehall/pull/6793/files#diff-e1322c398ec3f47afd61ed339782f4560e6c064496c6956e9c046296b29ad064R21).

We can do it if you feel strong about it but it's just the same code copied and pasted.

Reply:
This is how the page would look if we dismantle it:

```ruby
<% content_for :page_title, t('admin.whats_new.title') %>
<% content_for :title, t('admin.whats_new.title') %>
<% content_for :title_margin_bottom, 0 %>

<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
    <p class="govuk-body-l"><%= t('admin.whats_new.summary') %></p>

    <p class="govuk-body app-view-whats-new__last-updated"><%= t('admin.whats_new.last_updated') %></p>

    <div id="content-list">
      <%= render "govuk_publishing_components/components/contents_list", {
        contents: [
          {
            href: "##{t('admin.whats_new.introduction.heading').parameterize(separator: '-')}",
            text: t('admin.whats_new.introduction'),
          },
          {
            href: "##{t('admin.whats_new.recent_changes.heading').parameterize(separator: '-')}",
            text: t('admin.whats_new.recent_changes'),
          },
          {
            href: "##{t('admin.whats_new.guidance.heading').parameterize(separator: '-')}",
            text: t('admin.whats_new.guidance'),
          }
        ]
      } %>
    </div>

    <%= render partial: "admin/whats_new/section", locals: t('admin.whats_new.introduction') %>
    <%= render partial: "admin/whats_new/section", locals: t('admin.whats_new.recent_changes') %>
    <%= render partial: "admin/whats_new/section", locals: t('admin.whats_new.guidance') %>
  </div>
</div>
```

Reply:
I personally don't see the value but open for my view to be changed

Reply:
Ah I think I misunderstood a bunch of it. 

To be honest your refactor looks way easier to understand than the previous version so I'd go with that. 

Rather than having just one section partial that does two jobs (govspeak and updates) it'd be better if that was separated into distinct ones, that'd make it easier to understand. It also seems like the updates code is fully specific to recent changes, so that'd me more explicit with a dedicated partial.

Reply:
Done